nft-client/src/main.rs

```diff
@@ -504,18 +503,17 @@ async fn main() -> anyhow::Result<()> {
             )
             .context("Could not parse the accounts file.")?;
 
-            let owner = convert_account_address(&account_data.address);
             let mint_parameter = MintParams::new(
-                concordium_contracts_common::Address::Account(owner),
+                concordium_contracts_common::Address::Account(account_data.address),
```

Comment:
I recommend implementing
```rust
From<AccountAddress> for Address
```
and likewise for ContractAddress.

Then using `.into` here.